ajax/libs/pagecss/package.json

```diff
@@ -23,7 +20,11 @@
 			"files": [
 				"**/*"
 			]
-		}],
+		}],	
+		"keywords": [
+		"pagecss",
+		"css",
```

Comment:
also, keywords should be outside autoupdate section, the origin place was the correct place.